config/routes.rb

```diff
@@ -7,12 +7,12 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
-  
+
   namespace :api do
     namespace :v1 do
       resources :trips do
         resources :activities, only: [:index]
-        resources :accomodations, only: [:show]
+        resources :accommodations, only: [:create, :update]
       end
```

Comment:
I was thinking that instead of having a route to `:accommodations, only: [:show]`, we could add a relationship to Trips, and let it appear there? Less querying and logic needed in our controller, just to get an accommodation that belongs to Trip

Reply:
I don't think many accommodations attributes get passed if we use the `has_many :accommodations` in our trips serializer. We might be able to get the accommodation `:id` and the type.

Reply:
You're right!